substratest/factory.py

```diff
@@ -429,7 +429,7 @@ def __init__(self, name):
         self._dataset_counter = Counter()
         self._objective_counter = Counter()
         self._algo_counter = Counter()
-        self._workdir = pathlib.Path(tempfile.mkdtemp())
+        self._workdir = pathlib.Path(tempfile.mkdtemp(prefix=os.getcwd() + '/'))
```

Comment:
Is it absolutely necessary to work in the CWD?

Reply:
The error if I add nothing is:
```
E       docker.errors.APIError: 500 Server Error: Internal Server Error ("b'Mounts denied: \r\nThe path /var/folders/k2/kkx936g94tn2y6xtrdf8cj_m0000gn/T/tmpbl4cf2dv/dataset-0/opener.py\r\nis not shared from OS X and is not known to Docker.\r\nYou can configure shared paths from Docker -> Preferences... -> File Sharing.\r\nSee https://docs.docker.com/docker-for-mac/osxfs/#namespaces for more info.\r\n.'")
```
For the Finder, /var/folders does not exist and without both /var/folders and /private/var/folders added to the shared files we get this error.

--> We could use `/tmp/` instead

Reply:
👍